### PR TITLE
Convert Aircraft.Facing to WAngle.

### PR DESCRIPTION
**Pull request by pchote**
Another step on the path towards higher-resolution facings. Related to #18058, #18067, #18211.

This PR converts `Aircraft.Facing` which can be relatively cleanly separated from the more general `IFacing.Facing`.

### CODE REVIEW

**Review by Mailaender — APPROVED** [review]
Can't spot regressions in code nor in-game.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/726fb136af004889c5df28a4ffcd3fb999e2ac9f...142cad62bc1977445f1a1ef9314f87cd2c514bbf

**Comment by reaperrr** [comment]
Damn, actually overlooked one regression:

At least when landing, aircraft (transport heli in RA) seem to turn 4x as fast as before.

**Comment by pchote** [comment]
Good catch - that will be fixed in the next PR.

File: OpenRA.Mods.Common/Activities/Air/Land.cs [hunk 1]
```diff
@@ -60,8 +60,8 @@ public Land(Actor self, Target target, WDist landRange, WVec offset, int facing
 
 			// NOTE: desiredFacing = -1 means we should not prefer any particular facing and instead just
```
Comment (tovl) [inline OpenRA.Mods.Common/Activities/Air/Land.cs:61]: This comment should be updated.

**Pull request merged**